app/components/ui/images.tsx

```diff
@@ -11,5 +11,19 @@ type ImageProps = ImgHTMLAttributes<HTMLImageElement> &
   };
 
 export function Image({ ...props }: ImageProps) {
-  return <UnpicImage {...props} />;
+  const [loaded, setLoaded] = useState(false);
+
+  return (
+    <div className="relative">
+      {!loaded && <div className="absolute opacity-5 animate-pulse bg-slate-100 w-full h-full" />}
```

Comment:
During the PR review, it was suggested that we can add a prop to customize the shape of the loading div. This can be a future improvement!